front/src/applications/operationalStudies/hooks/useLazyLoadTrains.ts

```diff
@@ -43,8 +43,7 @@ const useLazyLoadTrains = ({
   trainSchedules,
   setTrainIdsToProject,
 }: UseLazyLoadTrainsProps) => {
-  const { getElectricalProfileSetId } = useOsrdConfSelectors();
-  const electricalProfileSetId = useSelector(getElectricalProfileSetId);
+  const electricalProfileSetId = useSelector(getOperationalStudiesElectricalProfileSetId);
```

Comment:
This one is tricky, but this hook is used actually in stdcm too

Reply:
Oh, good catch :) I'll move `electricalProfileSetId` to the props, then.

Reply:
Fixed